documentation/docs/guide/setting/Using-BCGov-Block-Theme.md

```diff
@@ -19,6 +19,40 @@ Some features will need to be turned on to become available to the admin environ
 
 Other settings for admin enabling options in the Block Theme can be found on the  **BCGov Block Theme Settings**  page
 
+### Custom Notice Banner
```

Comment:
i think this documentation already exists, can you verify, https://bcgov.github.io/bcgov-wordpress-block-theme/guide/setting/Notice-Banner.html

Reply:
It's summary information in the Quick Start area so I think it could be helpful. How about the addition of a direct link:
<img width="786" alt="image" src="https://github.com/bcgov/bcgov-wordpress-block-theme/assets/5200460/3c82c4ce-271f-452d-af56-3a44da77a6a7">